### PR TITLE
Show worker also when job already done

### PR DESCRIPTION
**Pull request by Martchus**
See https://progress.opensuse.org/issues/6564

### CODE REVIEW

File: t/ui/14-dashboard-parents.t [hunk 1]
```diff
@@ -72,7 +72,7 @@ my $element = $driver->find_element('opensuse', 'link_text');
 ok($element->is_displayed(), 'link to child group expanded');
 
 # first try of click does not work for unknown reasons
-for (0 .. 7) {
+for (0 .. 10) {
```
Comment (Martchus) [inline t/ui/14-dashboard-parents.t]: Not sure why this must be increased now.

**Comment by Martchus** [comment]
The coverage check doesn't seem to consider calls from templates in PhantomJS test (maybe because PhantomJS runs in forked process).

**Comment by okurz** [comment]
LGTM

**Comment by aaannz** [comment]
As you are doing another migration, can you cherry-pick https://github.com/aaannz/openQA/commit/20ae6373dcdf5b91aab1382e35a8598dd091f836 to fix system user creation?

**Comment by Martchus** [comment]
Ok, I'll cherry pick.

**Pull request merged**